README.md

```diff
@@ -63,6 +63,8 @@ Note that the external-attacher does not scale with more replicas. Only one exte
 
 * `--worker-threads`: The number of goroutines for processing VolumeAttachments. 10 workers is used by default.
 
+* `--max-entries`: The max number of entries per page for processing ListVolumes. No limits (0 entries) is used by default.
```

Comment:
Please be more specific about what `0` means. Like "0 means no limit and it is the default value".

Reply:
Done